action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -60,20 +58,18 @@ type (
 		preimageSnapshot map[int]preimageMap
 		dao              db.KVStore
 		cb               db.CachedBatch
-		hu               config.HeightUpgrade
+		isPreAleutian    bool
```

Comment:
suggest to keep hu, we may have a future height change that would affect StateDBAdapter behavior

Reply:
i just recalled, HeightUpgrade is created to address the wrong log topic at Aleutian height, b/c we think pass a bool "isPreAleutian" is not a general good solution
so better keep hu

Reply:
rename it to notFixTopicCopyBug